### ISSUE TITLE
Why are you wasting GitHub

### ISSUE THREAD

**Issue by GNOME-IS-LIFE** [kelseyhightower/nocode#1987]

**Comment by GNOME-IS-LIFE** [comment]
shit pressed the enter key by mistake

**Comment by GNOME-IS-LIFE** [comment]
GitHub is hosting lottsa code, and their servers are being wated on this hot trash xD

**Comment by Pizzacus** [comment]
That's the point!

By writing no code, we save them a lot of precious data space!!!

**Comment by GNOME-IS-LIFE** [comment]
What about the documentation?
that wastes more space than I am in the universe

**Comment by Somefive** [comment]
Instead, I think this repo could be moved to [this place](about:blank).

**Comment by zLinz** [comment]
GitHub's server doesn't mind. why do you mind?

**Comment by lewisdonovan** [comment]
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi velit elit, condimentum in turpis non, facilisis malesuada lorem. Etiam ligula nisi, convallis fermentum dolor iaculis, vulputate feugiat sapien. Duis aliquet quis magna in ullamcorper. Pellentesque fermentum elit vel nisl pharetra, vel semper ligula tincidunt. Nam id viverra tortor, id viverra neque. Nulla pretium nunc ac ipsum suscipit aliquam. In consectetur pretium sapien, vitae rhoncus ligula lacinia ac. Nam leo eros, tempus eu metus id, rutrum gravida nibh. Sed eu eleifend arcu.

Sed ac ultricies lorem. Maecenas id nibh nec eros congue aliquet. Pellentesque enim tellus, hendrerit a sem a, euismod cursus mauris. Nullam eleifend libero in tellus eleifend, commodo fringilla augue ullamcorper. In tincidunt volutpat nisi, vel aliquet sapien dignissim quis. Morbi quis ante mi. Duis luctus nisi lectus, sed gravida enim egestas vitae.

Duis vel sapien non turpis tempus rhoncus. Sed velit ipsum, vehicula sed sapien a, iaculis aliquet massa. Donec quis blandit mi. Quisque placerat ligula vitae dapibus porttitor. Quisque et urna nec ex mattis fringilla. Proin mollis consequat augue congue volutpat. Nunc metus nibh, dapibus non ligula eu, venenatis interdum tortor. Duis gravida lectus et fermentum sagittis. Sed pharetra nibh viverra varius ultricies. Maecenas eu odio dui. Nunc purus odio, efficitur in aliquet id, molestie non erat.

Fusce at nunc imperdiet libero feugiat venenatis vitae eu ex. Proin malesuada tempor tortor, nec malesuada nibh placerat sed. Cras non tortor nibh. Suspendisse eleifend nunc sollicitudin, efficitur elit vitae, rhoncus ipsum. Nulla ut dolor nisi. Mauris congue leo odio, in pulvinar odio vulputate eu. Integer efficitur porttitor sapien lobortis blandit. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis maximus ac ex ac sollicitudin. Aenean ut nibh nec mi aliquet lobortis id at purus.

Donec eu quam mollis, pellentesque magna id, interdum lectus. Sed pretium placerat lacinia. Maecenas eu accumsan ante, vitae sodales purus. Sed pulvinar orci sed velit faucibus vulputate. Cras pharetra sapien eget posuere consequat. Nulla magna tellus, porttitor mattis augue eu, fermentum aliquet nunc. Aenean et mollis dui. Fusce id posuere urna. Suspendisse nisl tortor, feugiat a nisi ac, venenatis vehicula sapien. Duis in sapien in augue fringilla sagittis. Mauris laoreet tortor lorem, a auctor ex vulputate eu. Pellentesque aliquam pellentesque ligula, ut tristique orci condimentum ac. Vestibulum convallis vitae nisi a dignissim. Etiam scelerisque malesuada nibh in consectetur.

**Comment by 06chaynes** [comment]
You're right, the load might be too much... we should replicate nocode to Bitbucket and Gitlab too. Just to be safe.
